src/lib/meilisearch.ts

```diff
@@ -253,14 +256,68 @@ class MeiliSearch {
   ///
 
   /**
-   * Get private and public key
+   * Get all API keys
    * @memberof MeiliSearch
    * @method getKey
-   * @returns {Promise<Keys>} Promise containing an object with keys
+   * @returns {Promise<Keys>} Promise returning an object with keys
```

Comment:
It's returning an array of object key?

Reply:
Yes its an array for the moment but later it should be an object with the `results` key